samples/bluetooth/central_autoconn/README.rst

```diff
@@ -0,0 +1,27 @@
+.. _bluetooth_central_autoconn:
+
+Bluetooth: Central Autoconnect
+##################
+
+Overview
+********
+
+Application demonstrating BLE Central role with whitelist based autoconnect
+functionality by scanning for other BLE devices and establishing a connection
+to the first one with a strong enough signal. Autoconnect is enabled so that if
```

Comment:
how is  "strong enough signal" defined?  (I added something to the previous comment).